packages/core-plugin-api/api-report.md

```diff
@@ -632,14 +645,39 @@ export type PluginConfig<
   routes?: Routes;
   externalRoutes?: ExternalRoutes;
   featureFlags?: PluginFeatureFlagConfig[];
+  info?: PluginConfigInfo;
   __experimentalConfigure?(options?: PluginInputOptions): {};
 };
 
+// @public
+export type PluginConfigInfo =
+  | Partial<PluginInfo>
+  | (Omit<Partial<PluginInfo>, 'packageJson'> & {
+      packageJson: LazyLoadedPackageJson;
+    })
+  | LazyLoadedPackageJson;
+
 // @public
 export type PluginFeatureFlagConfig = {
   name: string;
 };
 
+// @public
+export type PluginInfo = {
+  packageJson?: Record<string, unknown>;
+  description?: string;
+  version?: string;
+  ownerEntityRef?: string;
+  links: PluginInfoLink[];
+  role?: string;
```

Comment:
Would this ever be anything else than `frontend-plugin`?

Reply:
For now, perhaps not. Although perhaps a _common_ plugin.

Reply:
Common plugins don't exist though, it's just frontend of backend, the rest are libraries that don't export anything that the framework is aware of. That's the current state at least x)

Reply:
Ok, removed.